src/services/picture.ts

```diff
@@ -34,4 +38,55 @@ export class PictureService {
       upload.start();
     });
   }
+
+  convertURLtoBlob(URL: string): Promise<Blob> {
+    return new Promise((resolve, reject) => {
+      const image = document.createElement('img');
+
+      image.onload = () => {
+        try {
+          const dataURI = this.convertImageToDataURI(image);
+          const blob = this.convertDataURIToBlob(dataURI);
+
+          resolve(blob);
+        }
+        catch (e) {
+          reject(e);
+        }
+      };
+
+      image.src = URL;
+    });
+  }
+
+  convertImageToDataURI(image: HTMLImageElement): string {
+    // Create an empty canvas element
+    const canvas = document.createElement('canvas');
```

Comment:
Maybe use a package for that? something like https://www.npmjs.com/package/datauri / https://www.npmjs.com/package/image-to-data-uri